core/pkg/opencost/summaryallocation.go

```diff
@@ -40,6 +40,7 @@ type SummaryAllocation struct {
 	ExternalCost           float64               `json:"externalCost"`
 	Share                  bool                  `json:"-"`
 	UnmountedPVCost        float64               `json:"-"`
+	Efficiency             float64               `json:"efficiency"`
```

Comment:
Is this necessary in the SummaryAllocation struct? What was the issue with the previous `sa.TotalEfficiency()`?

Reply:
It's a good point and worth discussing. We moved the calculation of this to the DB so that it can be sorted. That might not cover us for OC though. I added the field in here so that the DB and the go code can be on the same page and not do the same calculation in 2 places. I will implement a fallback mechanism, wait one